testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -151,6 +151,13 @@ def test_toggle_atom_nomatch_mismatch_atoms(self, universe, reference):
         with pytest.raises(SelectionError):
             align.alignto(u, ref, select='all', match_atoms=False)
 
+    def test_no_atom_masses(self, universe, reference):
```

Comment:
You don't use `reference` in this test. That being said, it might be worth adding a test where one universe has masses and the other one doesn't.

Reply:
Also, planning to add this function:
`def test_one_atom _has_masses(self, universe, reference,selection="protein and backbone"):
        #if only atom has _has_masses
        u=mad.Universe.empty(6, 2, atom_resindex=[0, 0, 0, 1, 1, 1], trajectory=True)
        ref = reference.select_atoms(selection).atoms[1:]
        with pytest.warns(SelectionWarning):
            align.get_matching_atoms(u.atoms, ref.atoms)`

Reply:
Also. planning to add this function, Please tell if any changes are required:
![Screen Shot 2020-03-21 at 6 33 07 PM](https://user-images.githubusercontent.com/45939449/77227001-85b00200-6ba2-11ea-84a7-ca1ab365a813.png)

Reply:
Try adding it to your tests, running them locally, and seeing what error messages come up.

Reply:
```>python
with pytest.raises(TypeError) as exc:
        fit_translation(test_u, ref_u, weights="mass")(ts)
        assert 'atoms.masses is missing' in str(exc.value)
```
Will adding the above finction to `test_fit.py` do the job?